src/Test/Utilities/Portable/CommonTestBase.cs

```diff
@@ -472,7 +472,7 @@ private void AddReferencedCompilations(IEnumerable<Compilation> referencedCompil
 
         #region IL Verification
 
-        internal abstract string VisualizeRealIL(IModuleSymbol peModule, CompilationTestData.MethodData methodData, IReadOnlyDictionary<int, string> markers);
+        internal abstract string VisualizeRealIL(IModuleSymbol peModule, CompilationTestData.MethodData methodData, IReadOnlyDictionary<int, string> markers, Cci.IMethodBody methodBody);
```

Comment:
Instead of passing the method body, why not just pass the boolean for `LocalsAreZeroed`? Do we need the method body for anything else?

Reply:
You have a point.